internal/util/update/update.go

```diff
@@ -105,6 +104,9 @@ type Command struct {
 	// Path is the filepath to the local package
 	Path string
 
+	// FullPackagePath is the absolute path to the local package
+	FullPackagePath string
```

Comment:
Why do we need this path ?

Reply:
The value of `Path` is currently the relative path to the local package. This contains the absolute path. The goal is to transition the implementation of the update command and all the strategies to use the full path in `FullPackagePath` instead of the relative one. I want to keep both around while transitioning over since it will require quite a few changes across the four different strategies.